.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can you add this file to the `.gitignore` file? We will also need to make sure it gets removed from this PR. Can't remember off the top of my head how to do this! But maybe something like `git rm <filename>`? https://stackoverflow.com/questions/1139762/ignore-files-that-have-already-been-committed-to-a-git-repository